cmake/VcpkgToolchain.cmake

```diff
@@ -61,9 +61,9 @@ function(_plasma_vcpkg_setup_binarycache)
     endif()
 endfunction()
 
-# Binarycache can only be used on Windows or if mono is available. Since we are not using
-# vcpkg on CI, let's just disable this altogether.
-if(_HOST_IS_WINDOWS)
+# Binarycache can only be used on Windows or if mono is available.
+find_program(_VCPKG_MONO mono)
```

Comment:
The only challenge here is that this adds a cache entry called `_VCPKG_MONO` that implies is is editable. Of course, vcpkg itself does the same thing with `cl.exe` in `_VCPKG_CL`, so maybe I shouldn't complain.

Reply:
Yeah, this was based on what I saw it doing with `_VCPKG_CL`